data/brands/amenity/fuel.json

```
@@ -1096,8 +1096,8 @@
       "tags": {
         "amenity": "fuel",
         "brand": "Conoco",
-        "brand:wikidata": "Q214763",
-        "brand:wikipedia": "en:ConocoPhillips",
+        "brand:wikidata": "Q1126518",
+        "brand:wikipedia": "en:Conoco",
```

I split out [Q109341187](https://www.wikidata.org/wiki/Q109341187) for the Conoco brand as it has existed under multiple owners over the years and refocused [Q1126518](https://www.wikidata.org/wiki/Q1126518) to be about Conoco Inc. specifically. [The English Wikipedia article](https://en.wikipedia.org/wiki/Conoco) is an odd mixture of the two Conocos, so I linked it to the new item instead of the old one. The other Wikipedias only have articles that focus solely on Conoco Inc., so I left those sitelinks alone.

```suggestion
        "brand:wikidata": "Q109341187",
        "brand:wikipedia": "en:Conoco",
```

---

Phillips 66 should probably get the same treatment at some point.